lambda/controller/create-replica/index.js

```diff
@@ -24,26 +24,42 @@ exports.handler = function(event, context, callback){
       return callback(err);
     }
 
+    var billingMode = 'PROVISIONED';
+
+    // Check the billing mode of the table
+    if (data.Table.hasOwnProperty('BillingModeSummary')){
```

Comment:
Minor: Mostly a style thing but you can use `&&` to chain logic in an if block. So 

```
if (data.Table.hasOwnProperty('BillingModeSummary')){
  if (data.Table.BillingModeSummary.BillingMode == 'PAY_PER_REQUEST') {
```

becomes 

```
if (data.Table.hasOwnProperty('BillingModeSummary') && 
    data.Table.BillingModeSummary.BillingMode == 'PAY_PER_REQUEST') {
```

If the part before the `&&` returns false the second part isn't checked.

Reply:
👍

Reply:
@pwilder-sig Hmmm...your suggestion didn't work here.

Reply:
That's unexpected. I can come take a look if you want to keep trying but it's honestly not *that* important.

Reply:
Well now I'm curious why...  :)